moc-monitoring/base/alert-manager/deployment.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: alertmanager
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+      - name: alertmanager
+        image: quay.io/prometheus/alertmanager:v0.25.0
+        args:
+        - "--config.file=/etc/alertmanager/alertmanager.yaml"
+        - "--storage.path=/alertmanager"
+        ports:
+        - containerPort: 9093
```

You don't need to change this, but it's a good idea to give your ports a name:

```suggestion
        - name: alertmanager
          containerPort: 9093
```

Because then you can write your service like this:

```
apiVersion: v1
kind: Service
metadata:
  name: alertmanager
spec:
  ports:
  - port: 9093
    targetPort: alertmanager
```

Which is nice because the service doesn't need to be updated if you change the `containerPort`.